packages/frontend/app/lib/api/asset.server.ts

```diff
@@ -27,9 +27,10 @@ import type {
   WithdrawAssetLiquidity,
   WithdrawAssetLiquidityVariables
 } from '~/generated/graphql'
-import { apolloClient } from '../apollo.server'
+import { getApolloClient } from '../apollo.server'
 
-export const getAssetInfo = async (args: QueryAssetArgs) => {
+export const getAssetInfo = async (request: Request, args: QueryAssetArgs) => {
+  const apolloClient = await getApolloClient(request)
```

Comment:
example of changs to apollo client to support requestor specific api signature and tenant id header as described here: https://github.com/interledger/rafiki/pull/3213/files#r1916826388